### PR TITLE
Some updates of deps

### PR DESCRIPTION
**Pull request by tertsdiepraam**
Some updates that renovate had issues with because of `cargo deny`.

### CODE REVIEW

File: deny.toml [hunk 1]
```diff
@@ -80,8 +83,6 @@ skip = [
   { name = "windows_x86_64_gnullvm", version = "0.42.2" },
   # windows-targets
   { name = "windows_x86_64_msvc", version = "0.42.2" },
-  # cpp_macros
-  { name = "aho-corasick", version = "0.7.20" },
```
Comment (cakebaker) [inline deny.toml:84]: Why do you remove `aho-corasick`? It's makes the CI fail because we use two versions of it.

Comment (tertsdiepraam) [reply deny.toml:84]: Strange, I got a warning that the rule wasn't used. I put it back.

**Comment by cakebaker** [comment]
Why do you include `clap` but not `tempfile`? There are open `renovate` PRs for `is_terminal` and `tempfile` because of `cargo-deny` and `rustix`. There is no `renovate` issue with `clap`.

File: deny.toml (hunk 1 above)
Comment (cakebaker) [reply deny.toml:84]: You have to run `cargo deny --all-features check`.

Comment (tertsdiepraam) [reply deny.toml:84]: Oh I forgot about that

**Comment by tertsdiepraam** [comment]
I must've gotten tempfile and some other name confused. I added it. I did clap to anticipate a renovate PR, because it is out of date (a bit).

**Comment by cakebaker** [comment]
We could specify a patch version for `clap` in `Cargo.toml`, then we would get more `renovate` PRs.

**Comment by tertsdiepraam** [comment]
I think that's unrelated. Renovate works on Cargo.lock versions as well. We just have a limit on Renovate PRs I think

**Pull request merged**